src/antares/model/settings/antares_simulation_parameters.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+from typing import Any, Dict, Optional
+
+from pydantic import BaseModel, Field
+
+from antares.model.settings.solver import Solver
+
+
+class AntaresSimulationParameters(BaseModel):
+    solver: Solver = Solver.SIRIUS
+    nb_cpu: Optional[int] = None
+    unzip_output: bool = Field(True, alias="auto_unzip")
+    output_suffix: Optional[str] = None
+    presolve: bool = False
+
+    @property
+    def other_options(self) -> str:
+        options = []
+        if self.presolve:
+            options.append("presolve")
+        if self.solver != Solver.SIRIUS:
+            options.append(self.solver.name.lower())
+        return " ".join(options)
+
+    def model_dump(self, *args: Any, **kwargs: Any) -> Dict[str, Any]:
```

Comment:
We should rename this method `to_api()` instaed of model_dump because when we'll implement the local method we'll have to parse the object differently